tests/centosci/sink-clustered-deployment.sh

```diff
@@ -34,7 +34,7 @@ IMG="${CI_IMG_OP}" make test
 
 # Run integration tests
 SMBOP_TEST_CLUSTERED=1 SMBOP_TEST_EXPECT_MANAGER_IMG="${CI_IMG_OP}" \
-	./tests/test.sh
+	./tests/test.sh || ./tests/post-test-info.sh
```

Comment:
But this would make overall test to be marked as successful. Don't we need to report CI run as failed?